projects/mobile-http/src/lib/types.ts

```diff
@@ -23,6 +23,8 @@ export class CordovaHttpClientOptions {
   headers: KeyValuePair<string> = {};
   responseType: string = 'json';
   data: any;
+  serializer?: string;

```

Comment:
Remove optional tag from this model.
```suggestion
  serializer: string;
```